src/grt/README.md

```diff
@@ -228,6 +228,60 @@ The `global_route_debug` command allows you to start a debug mode to view the st
 
 ## Limitations
 
+## Using the Python interface to grt
+
+**NOTE:** The Python interface is currently in development and may
+change.
+
+This api tries to stay close to the api defined in C++ class
+GlobalRouter that is located in grt/include/grt/GlobalRouter.h. 
+
+When initializing a design, a sequence of Python commands might look like
+the following:
+
+    from openroad import Design, Tech
+    tech = Tech()
+    tech.readLef(...)
+    design = Design(tech)
+    design.readDef(...)
+    gr = design.getGlobalRouter()
+    
+Here are some options / configurations to the globalRoute
+command. (See GlobalRouter.h for a complete list)
+
+    gr.setGridOrigin(x, y)                     # int, default 0,0
+    gr.setCongestionReportFile(file_name)      # string
+    gr.setOverflowIterations(n)                # int, default 50
+    gr.setAllowCongestion(allowCongestion)     # boolean, default False
+    gr.setCriticalNetsPercentage(percentage)   # float
+    gr.setMinRoutingLayer(minLayer)            # int
+    gr.setMaxRoutingLayer(maxLayer)            # int
+    gr.setMinLayerForClock(minLayer)           # int
+    gr.setMaxLayerForClock(maxLayer)           # int
```

Comment:
It would be better if these took a dbLayer* rather than an integer.

Reply:
So these are defined by the current C++ api (in GlobalRouter.h) that is then used by GlobalRouter.i and then further wrapped by GlobalRouter.tcl. So if we change this in C++ we will have to change the tcl interface too (or at least the tcl code to maintain a similar interface), or we can have a parallel interface (not my preference).